sentinelsat/sentinel.py

```diff
@@ -1427,33 +1427,42 @@ def _parse_odata_response(product):
     return output
 
 
-def placename_to_wkt(placename):
-    """Geocodes the placename to rectangular bounding extents using Nominatim API and
+def placename_to_wkt(place_name):
+    """Geocodes the place name to rectangular bounding extents using Nominatim API and
        returns the corresponding 'ENVELOPE' form Well-Known-Text.
 
     Parameters
     ----------
-    placename : str
+    place_name : str
         the query to geocode
 
+    Raises
+    ------
+    ValueError
+        If no matches to the place name were found.
+
     Returns
     -------
-    full name and coordinates of the queried placename
-        list in the form [wkt string in form 'ENVELOPE(minX, maxX, maxY, minY)', string placeinfo]
+    wkt_envelope : str
+        Bounding box of the location as an 'ENVELOPE(minX, maxX, maxY, minY)' WKT string.
```

Comment:
A bounding box and an evenlope *can* be identical but don't have to be. Can we use a single terminology throughout the codebase?

The Envelope vs. Extent definitions can be rather confusing, see for instance PostGIS [ST_Envelope](https://postgis.net/docs/ST_Envelope.html) vs. [ST_Extent](https://postgis.net/docs/ST_Extent.html) (which I find to be a bad pattern).

Reply:
I guess there can be, but I don't see any way they could have a different meanings in our use cases?
Either way, I personally prefer "bounding box" as it sounds a bit clearer to me.

Reply:
Let's go with that then :+1: